docs/layouts/shortcodes/source-integration/validate.md

```diff
@@ -22,7 +22,7 @@ If you see a message that the webhook wasn't added, add one manually.
 {{ else if eq $source "GitLab" }}
   {{ $reqdPerms := "Maintainer or Owner" }}
   {{ $permsLink = "https://docs.gitlab.com/ee/user/permissions.html" }}
-{{ end }} 
+{{ end }}
```

Comment:
@chadwcarlson And just above this line the one that doesn't work. Keeps displaying "Admin" on the GitLab page.